iina/PlayerCore.swift

```diff
@@ -1089,6 +1086,11 @@ class PlayerCore: NSObject {
   func playbackRestarted() {
     Logger.log("Playback restarted", subsystem: subsystem)
     reloadSavedIINAfilters()
+
+    if info.vid == 0 {
```

Comment:
When deciding whether we will show the default album art, `self.info.videoTracks.isEmpty` is used. Not sure which one is the better one, or should we make it a function in `PlaybackInfo`?